WeakAuras/WeakAuras.lua

```diff
@@ -2850,6 +2850,24 @@ function WeakAuras.Modernize(data)
     end
   end
 
+  -- Version 14 was introduced March 2019 in BFA
+  if data.internalVersion < 14 then
+    if data.triggers then
+      for triggerId, triggerData in pairs(data.triggers) do
+        if type(triggerData) == "table"
```

Comment:
If the triggerData isn’t a table, then we have bigger problems :)

Reply:
it's for filtering "disjunctive" and "activeTriggerMode"

Reply:
Ah, fair. ipairs would work then.